app/views/desks/show.html.erb

```diff
@@ -1,7 +1,28 @@
-<%= link_to "Back to desk list", root_path %>
-<h3>Desk details</h3>
-<ul>
-  <li><%= @desk.id %></li>
-  <li><%= @desk.name %></li>
-  <li><%= @desk.occupied %></li>
-</ul>
+<nav aria-label="breadcrumb" role="navigation">
+  <ol class="breadcrumb">
+    <li class="breadcrumb-item"><a href="<%= root_path %>">Desk Listing</a></li>
```

Comment:
Use `link_to` instead of explicit anchor tags here as well

Reply:
When I tried using link_to, it was messing up the bootstrap styling, but maybe I did it wrong?

Reply:
Got it work, I was making a noobie mistake